### ISSUE TITLE
Stat 547 HW6 ready for grading

### ISSUE THREAD

**Issue by wswade2** [wswade2/Stat547#1]
@vincenzocoia @gvdr @ksedivyhaley @joeybernhardt @mynamedaike @pgonzaleze @derekcho

**Comment by jmurthy12** [comment]
Hi @wswade2,

**README**:
•	Your README.md is easy to navigate
•	You have provided the link for the hw06 folder markdown file of your assignment.

**Data Wrangling Wrapup**
•	Perhaps a descriptive title for your markdown file/hw06 folder such as “hw06_data_wrangling_wrap_up.md”/ hw06_data_wrangling_wrap_up.

**Writing Functions**
•	Your explanations on “for” loop helped me a lot to understand shall try to incorporate this in my next assignment.
•	Great usage of histogram and for loop logic. I got a lot to learn from this.

**Nested data frame**
•	Amazing work on Linear fit, Nested version of gapmider and colourful ggplots.
•	Perhaps you could explore more on Robust and quadratic fit
•	Your functions are nice and simple. 

**Process Report:**
•	You have provided lots of useful in you process report.
In summary, great job on the assignment with lots of useful information.

Thanks,
Janani.

**Comment by vmichalowski** [comment]
Hello @wswade2

I thought you did a great job on your homework 6! I just have a few comments:

- that's excellent that you came up with an original and useful function that you will likely use beyond this assignment!
- I found it easy to follow your code and thought-process.
- your plot of life expectancy versus country towards the end of the assignment was quite nice, especially with your use of colour - it communicated clearly which continents showed the higher expected life expectancies.
- I really liked your function for producing histograms in the first part of your assignment - I wonder if there was a way you could have extended that into your examination of nested data frames towards the end, but I'm guessing that the data set you used in part one was not nested, so perhaps it didn't lend itself easily to that

Overall, really great work on this assignment!

Victoria

**Comment by ksedivyhaley** [comment]
**At least two topics:** Yes (functions, nested data frame)
**Functions:** Partial (see comments)
**Nested data frame:** Ok (see comments)
**Reflection:** Yes

**Comments:**

- The code you develop in the first bit for plotting a bit of histograms is really neat! Unfortunately, you never actually define it as a function – which is odd because you do define a function in the second part (albeit a trivial one, le_vs_yr).
- Since you will be using this function, when you do define it I recommend adding error checking – at minimum make sure you're sending it a data frame.
- Your handling of the nested data frame is fine, but your graphs are not very informative since your “estimate” values represent estimates of both the slope and intercept of the linear model (see the “term” variable), which can't be compared the way you're trying to. Use filtering or reshaping first.

Your mark will be distributed later. If you would like more feedback, please feel free to message me on slack.
